### ISSUE TITLE
Episode 6 - plotting - needs better final example

### ISSUE THREAD

**Issue by brendam** [LibraryCarpentry/library-python#16]
Heading for final example reads:

`# A Realistic Example -- TODO`

The code in the example now works with the new data, but isn't really a useful plot. Need to create a better example.

**Comment by c-martinez** [comment]
What would be a good example? I could suggest plotting the number of authors per language to see if English authors tend to collaborate more/less than in other languages.

**Comment by brendam** [comment]
I've made a pull request #27 

I've stuck with numeric plotting examples. I also came up with this, but decided that it might be too messy to explain?

```
ax = articles_df.groupby('First_Author').Title.count().sort_values(ascending=False)[0:10].plot(kind='barh')
ax.set_xlabel('Number of articles')
```

**Comment by mkuzak** [comment]
Merged #27. Do you still want to add more examples? If not, you can close this.

**Comment by brendam** [comment]
happy as it is.
